Add docs for `HTMLLinkElement.integrity` and `HTMLScriptElement.integrity` (#33355)
mdn/content: files/en-us/web/html/element/link/index.md

- `integrity`
  - : Contains inline metadata — a base64-encoded cryptographic hash of the resource (file) you're telling the browser to fetch.

>     The browser can use this to verify that the fetched resource has been delivered free of unexpected manipulation.

Author: ```suggestion
    The browser can use this to verify that the fetched resource has been delivered without unexpected manipulation.
```
